backend/smoothie_pyserial.py

```diff
@@ -2,14 +2,20 @@
 
 import asyncio, json, math
 
+import atexit
+import sys
```

Comment:
Imports should be alphabetically ordered... (therefore sys should be a towards the end...

I'd also fix the orderings for all these imports...